schedule/jeos/sle/jeos-main.yaml

```diff
@@ -45,6 +45,7 @@ schedule:
     - console/check_network
     - console/system_state
     - console/consoletest_setup
+    - console/systemd_wo_udev
```

Comment:
Is this applicable to all versions? The feature request is only for 15-SP4 (and I guess Leap 15.4)

Reply:
oh, I have missed that! Thanks!

Reply:
yup,  I've just seen you did it for main_common.pm :)

Reply:
Ok, so the main test case is hot candidate to be migrated to main.pm schedule.